src/components/style-selector/medium-style-selector.vue

```diff
@@ -0,0 +1,50 @@
+<script setup lang="ts">
+import { useTranslation } from 'i18next-vue'
+import { computed } from 'vue'
+import { storeToRefs } from 'pinia'
+
+import { getDefaultMediumStyling } from '@/services/styles/styles.service'
+import MediumStyleItem from './medium-style-item.vue'
+import { StyleItem } from '@/composables/mvt-styles/mvt-styles.model'
+import { Layer } from '@/stores/map.store.model'
+import { useStyleStore } from '@/stores/style.store'
+
+const styleStore = useStyleStore()
+const { bgStyle } = storeToRefs(styleStore)
+
+const { t } = useTranslation()
+const props = defineProps<{
+  layer: Layer
+}>()
+const isColorVisible = computed(
+  () => props.layer.name === 'basemap_2015_global'
+)
+const styles = computed(
```

Comment:
Maybe add typing here, as two different types are possible ? Btw, I get typing errors on the `type-check` command here, I think.

Reply:
this confusion is being clarified in the next PR: https://github.com/Geoportail-Luxembourg/luxembourg-geoportail/blame/GSLUX-593_external_editor/src/services/styles/styles.service.ts#L8

it will be the same type

Reply:
ok